### DISCUSSION TITLE
ScriptReference/SystemInfo-unsupportedIdentifier

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#212] [Notes]
# ScriptReference/SystemInfo-unsupportedIdentifier

https://docs.unity3d.com/ScriptReference/SystemInfo-unsupportedIdentifier.html

**Comment by mvi** [comment]
This has a constant value of "n/a" see [here](https://github.com/Unity-Technologies/UnityCsReference/blob/master/Runtime/Export/SystemInfo.bindings.cs).
